networkit/profiling/profiling.py

```diff
@@ -13,12 +13,19 @@
 from . import multiprocessing_helper
 from . import stat
 from . import plot
+from ..support import MissingDependencyError
 
-from IPython.core.display import *
 import collections
 import math
 import fnmatch
 import random
+try:
+	import IPython
```

Comment:
Is this line necessary?

Reply:
You're right - it was not needed. I have removed it.